src/main/java/com/instana/operator/cache/Retry.java

```diff
@@ -0,0 +1,58 @@
+package com.instana.operator.cache;
+
+import io.reactivex.Scheduler;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.disposables.Disposables;
+
+import java.time.Duration;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class Retry {
```

Comment:
Wish we could use native Quarkus or Quartz here rather than rolling our own. But it probably doesn't make a lot of sense just for this. But a generic retry/recover pattern that integrates with Quarkus would be nice.

Reply:
If you know of one, would prefer to use that also! I didn't find one after a quick search though